sycl/source/detail/scheduler/commands.cpp

```diff
@@ -2033,6 +2038,22 @@ static pi_result SetKernelParamsAndLaunch(
     }
   };
 
+  for (auto &arg : Args) {
```

Comment:
Out of curiosity: can we also mark `arg` as `cosnt`? Though I assume it may not be easy as it will require adding `const` into several more places